src/components/SRNEntry.vue

```diff
@@ -147,7 +147,7 @@ export default {
         // either the user is valid or the user has been checked twice
         if (this.isUserValid || this.validateCount > 1) {
           if (
-            !redirectToDestination(
+            redirectToDestination(
               this.purposeParams,
               userID,
               this.redirectID,
```

Comment:
Just one doubt in this part, if due to some mistake the `purpose` in the query params is not either `plio` or `liveclass`, then the `redirectToDestination` method will return false. 

In that case, nothing will happen for the user right?
Should we show at least something? Like 404 not found or something?

Reply:
Yes we should. 404 for user. Error for us. Correct!

@sudheshna-donthineni  can you make sure you get this done in a separate PR?